src/libultra/os/resetglobalintmask.c

```diff
@@ -1,3 +1,8 @@
 #include "global.h"
 
-#pragma GLOBAL_ASM("asm/non_matchings/boot/resetglobalintmask/__osResetGlobalIntMask.s")
+void __osResetGlobalIntMask(u32 mask) {
+    register s32 prevInt = __osDisableInt();
```

Comment:
Is `register` actually needed here? :o

Reply:
Yeah, it actually changes the regalloc and stack usage significantly.